src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -128,6 +128,11 @@ public TagHelperContent AppendFormat(IFormatProvider provider, string format, pa
         /// <returns>A reference to this instance after the clear operation has completed.</returns>
         public abstract TagHelperContent Clear();
 
+        /// <summary>
+        /// Clears the <see cref="TagHelperContent"/> so it can be reused.
+        /// </summary>
+        public abstract void Reset();
```

Comment:
`Clear` nukes content and changes `IsModified` => `true`. `Reset` nukes content and turns `IsModified` => `false`.

`IsModified` is an indicator code gen uses to determine if a user doesn't want to run a tags content. [Example](https://github.com/aspnet/Razor/blob/d04f2fd2eeba3c085e9b0044731120f8f0f91492/test/Microsoft.AspNetCore.Razor.Test/TestFiles/CodeGenerator/Output/BasicTagHelpers.cs#L111-L114).